lighthouse-core/gather/drivers/driver.js

```diff
@@ -213,6 +221,75 @@ class Driver {
     });
   }
 
+  getServiceWorkerRegistrations() {
+    return new Promise(resolve => {
+      this.once('ServiceWorker.workerRegistrationUpdated', data => {
+        this.sendCommand('ServiceWorker.disable');
+        resolve(data);
```

Comment:
need to add what's been added to `getServiceWorkerVersions` to handle promise rejections.
```js
this.sendCommand('ServiceWorker.disable')
  .then(_ => resolve(data), reject);
```
and `this.sendCommand('ServiceWorker.enable').catch(reject);`